consensus/consensus.go

```diff
@@ -499,6 +513,12 @@ func (consensus *Consensus) DebugPrintValidators() {
 func (consensus *Consensus) UpdatePublicKeys(pubKeys []*bls.PublicKey) int {
 	consensus.pubKeyLock.Lock()
 	consensus.PublicKeys = append(pubKeys[:0:0], pubKeys...)
+	consensus.CommitteeAddresses = map[common.Address]bool{}
+	for _, pubKey := range consensus.PublicKeys {
+		consensus.CommitteeAddresses[utils.GetBlsAddress(pubKey)] = true
```

Comment:
if a validator not in the committee, does its key not exist here or value is false?

Reply:
Just not exist, there won't be false value. The map is used as a set